README.md

```diff
@@ -6,12 +6,14 @@ autoNumeric is a standalone Javascript library that provides live *as-you-type*
 <br>
 [![NPM version][npm-image]][npm-url]
 [![Downloads][downloads-image]][downloads-url]
+[![jsDelivr Hits][jsdelivr-image]][jsdelivr-url]
 [![Build Status][travis-image]][travis-url]
 [![Known Vulnerabilities][snyk-image]][snyk-url]
 [![Coverage Status][coveralls-image]][coveralls-url]
 <br>
 [![Gitter chat][gitter-image]][gitter-url]
 
+
```

Comment:
Thanks for using the link pattern.
Lastly, could you please remove this extra line? Thanks!